templates/visuals/.api/v1.12.0/schema.capabilities.json

```diff
@@ -667,6 +670,33 @@
             },
             "additionalProperties": false
         },
+        "tooltips": {
+            "type": "object",
+            "description": "Tells the host to include tooltips ability",
+            "properties": {
+                "supportedTypes": {
+                    "type": "object",
+                    "description": "Tells the host what tooltip types to support",
+                    "properties": {
+                    "default": {
+                        "type": "boolean",
+                        "description": "Tells the host to support showing default tooltips"
+                        },
```

Comment:
Indentation looks a bit off. Please run some re-formatter on the new section.

Reply:
Done